src/pf-dropdown/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "pf-dropdown",
+  "version": "1.0.0",
+  "description": "the pf-dropdown webcomponent",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
We should start implementing unit tests with new components. Otherwise, this will be left for someone else to pick up.